### PR TITLE
WIP - Transportation Storycard 001

### PR DESCRIPTION
**Pull request by jaronheard**

### CODE REVIEW

File: packages/2019-transportation/src/components/TransportationCard/index.js [hunk 1]
```diff
@@ -34,19 +32,17 @@ TransportationCard.displayName = "TransportationCard";
...
-  data: PropTypes.arrayOf(PropTypes.shape({})),
+  data: resourceShape,
   Layout: PropTypes.node
 };
 
 export default connect(
   state => ({
-    isLoading: isTransportationDataPending(state),
-    data: getTransportationData(state)
+    data: api.selectors.getDisturbanceStops(state) // for 2018 package, replace state with state.transportation2019
```
Comment (Dianna) [inline packages/2019-transportation/src/components/TransportationCard/index.js]: Note to self: check into state input methods

Comment (Dianna) [reply packages/2019-transportation/src/components/TransportationCard/index.js]: @jaronheard, my initial 2 cents: This looks like a fantastic tool to cut down on boilerplate when setting up new cards which makes creating new cards more accessible to devs who may not yet understand the ins and outs of redux. I think needing to customize how the state is input is a small price to pay for that and is something we can potentially abstract

**Comment by jaronheard** [comment]
@jaikamat @BJHunnicutt @Dianna 

I pushed some fixed to this branch, including a workaround to make the selector work off of the root state if it's running in the transportation2019 package, or off of the transportation2019 state if it's running in the 2018 package

**Comment by Dianna** [comment]
There are a few proptype issues here and there

**Comment by jaronheard** [comment]
@worc I've closed this PR, but feel free to reopen a separate one for this card

**Pull request closed without merging**